config/generic_domains.php

```diff
@@ -33,3 +33,826 @@
+    '.garden',
+    '.gb',
+    '.gbiz',
+    '.gd',
+    '.gdn',
+    '.gent',
+    '.gf',
+    '.gg',
+    '.ggee',
+    '.gi',
+    '.gift',
+    '.gifts',
+    '.gives',
+    '.gl',
+    '.glass',
+    '.gle',
+    '.global',
+    '.global',
+    '.globo',
+    '.gmail',
+    '.gmx',
+    '.goldpoint',
+    '.goo',
+    '.goog',
+    '.google',
+    '.gop',
+    '.gov',
+    '.gov',
+    '.gq',
+    '.graphics',
+    '.gratis',
+    '.green',
+    '.gripe',
+    '.gs',
+    '.guide',
+    '.guitars',
+    '.guru',
+    '.gy',
+    '.hamburg',
+    '.hangout',
+    '.haus',
+    '.healthcare',
+    '.help',
+    '.here',
+    '.hiphop',
+    '.hiv',
+    '.hk',
+    '.hk',
+    '.*.hk',
+    '.hm',
+    '.hn',
+    '.hn',
+    '.holdings',
+    '.holiday',
+    '.homes',
+    '.horse',
+    '.host',
+    '.host',
+    '.hosting',
+    '.house',
+    '.how',
+    '.hr',
+    '.ht',
+    '.hu',
+    '.ibm',
+    '.icu',
+    '.id',
+    '.*.id',
+    '.ie',
+    '.ifm',
+    '.il',
+    '.im',
+    '.immo',
+    '.immobilien',
+    '.in',
+    '.in',
+    '.inc',
+    '.industries',
+    '.infiniti',
+    '.info',
+    '.info',
+    '.ing',
+    '.ink',
+    '.ink',
+    '.institute',
+    '.insure',
+    '.int',
+    '.int',
+    '.international',
+    '.investments',
+    '.io',
+    '.iq',
+    '.ir',
+    '.ir',
+    '.is',
+    '.it',
+    '.java',
+    '.jcb',
+    '.je',
+    '.jetzt',
+    '.jobs',
+    '.joburg',
+    '.jp',
+    '.juegos',
+    '.kaufen',
+    '.kddi',
+    '.ke',
+    '.kg',
+    '.ki',
+    '.kim',
+    '.kitchen',
+    '.kiwi',
+    '.kiwi',
+    '.koeln',
+    '.kr',
+    '.kr',
+    '.krd',
+    '.kred',
+    '.ky',
+    '.kyoto',
+    '.kz',
+    '.la',
+    '.la',
+    '.lacaixa',
+    '.land',
+    '.lat',
+    '.latrobe',
+    '.lawyer',
+    '.lc',
+    '.lease',
+    '.leclerc',
+    '.legal',
+    '.lgbt',
+    '.li',
+    '.life',
+    '.lighting',
+    '.limited',
+    '.limo',
+    '.link',
+    '.live',
+    '.loans',
+    '.london',
+    '.london',
+    '.lotte',
+    '.lotto',
+    '.lt',
+    '.ltd',
+    '.ltda',
+    '.lu',
+    '.lu',
+    '.luxe',
+    '.luxury',
+    '.luxury',
+    '.lv',
+    '.eu.lv',
+    '.ly',
+    '.ly',
+    '.ma',
+    '.madrid',
+    '.maison',
+    '.management',
+    '.mango',
+    '.market',
+    '.marketing',
+    '.mc',
+    '.md',
+    '.me',
+    '.media',
+    '.meet',
+    '.melbourne',
+    '.meme',
+    '.memorial',
+    '.menu',
+    '.mg',
+    '.miami',
+    '.mil',
+    '.mil',
+    '.mini',
+    '.mk',
+    '.mk',
+    '.ml',
+    '.*.mm',
+    '.mn',
+    '.mn',
+    '.mo',
+    '.mobi',
+    '.moda',
+    '.monash',
+    '.money',
+    '.mortgage',
+    '.moscow',
+    '.motorcycles',
+    '.mov',
+    '.mp',
+    '.ms',
+    '.mtpc',
+    '.mu',
+    '.museum',
+    '.mx',
+    '.mx',
+    '.my',
+    '.my',
+    '.mz',
+    '.na',
+    '.nagoya',
+    '.name',
+    '.navy',
+    '.nc',
+    '.net',
+    '.net',
+    '.*.net',
+    '.*.net',
```

Comment:
This TLD is duplicated, but it could be changed to ".net.*" to support domains like ".net.pa".